starboard/nplb/media_can_play_mime_and_key_system_test.cc

```diff
@@ -72,6 +72,17 @@ DeviceType GetDeviceType() {
   return kDeviceTypeFHD;
 }
 
+testing::AssertionResult IsMimeAndKeySystemSupported(std::string param,
+                                                     std::string key_system) {
+  if (SbMediaCanPlayMimeAndKeySystem(param.c_str(), key_system.c_str()) ==
+      kSbMediaSupportTypeProbably) {
+    return testing::AssertionSuccess();
+  }
+  return testing::AssertionFailure()
+         << "SbMediaCanPlayMimeAndKeySystem(\"" << param << "\", \""
+         << key_system << "\") is unsupported.";
```

Comment:
It looks like it's also possible for there to be an AssertionFailure if the mime/keySystem is `kSbMediaSupportTypeMaybe`, so "is unsupported" may be a little misleading.

WDYT about something like:
`<< key_system << "\") is " << SbMediaCanPlayMimeAndKeySystem() << " but should be probably supported.";`

(If you do go this route then it would probably be good to store the `SbMediaCanPlayMimeAndKeySystem()` result in a local variable instead of call it twice.)

Reply:
Good suggestion, I tweaked the message a little.

Reply:
Thanks for making the change, looks good!